src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -981,4 +982,146 @@ public class CaseData extends Et1CaseData {
     private NoticeOfChangeAnswers noticeOfChangeAnswers9;
     @JsonProperty("changeOrganisationRequestField")
     private ChangeOrganisationRequest changeOrganisationRequestField;
+
+    //Respondent Tell Something Else
+    @JsonProperty("resTseSelectApplication")
+    private String resTseSelectApplication;
+    @JsonProperty("resTseVariableContent")
+    private String resTseVariableContent;
+    @JsonProperty("resTseDocument1")
+    private UploadedDocumentType resTseDocument1;
```

Comment:
Is there a particular reason why you are creating 12 individual fields rather than using a list?

Reply:
I'm not sure how a list helps us here. We're using 12 fields for 12 applications because the PageLabel needs to be different for each application, afaik we can't do this via callback population